README.md

```diff
@@ -844,6 +844,40 @@ export class ReactiveFormComponent implements OnInit {
   }
 }
 ```
+#### Template-driven example
+
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'template-driven-form',
+  template: `
+    <div>
+        <form (ngSubmit)="onSubmit()" #templateDrivenForm="ngForm" novalidate>
```

Comment:
Could you name the variable simply `#form` for brevity?

Reply:
I used a more verbose naming, because the relationship with the submit button is more understandable (in L907).